### ISSUE TITLE
C2039, C2143, C4430, C2061 Errors During Install

### ISSUE THREAD

**Issue by amylashley** [nvaccess/nvda#5532]
Hello- I'm trying to run scons source on my Windows 7 computer and receiving the following errors. Please help! 

c:\nvda\build\x86\remote\inProcess.h(14) : error C2039: 'function' : is not a me
mber of 'std'
c:\nvda\build\x86\remote\inProcess.h(14) : error C2143: syntax error : missing '
;' before '<'
c:\nvda\build\x86\remote\inProcess.h(14) : error C4430: missing type specifier -
 int assumed. Note: C++ does not support default-int
c:\nvda\build\x86\remote\inProcess.h(15) : error C2061: syntax error : identifie
r 'execInWindow_funcType'
scons: *** [build\x86\remote\injection.obj] Error

**Comment by jcsteh** [comment]
What version(s) of Visual Studio do you have installed? It sounds like 
you're trying to build NVDA with something earlier than VS 2012.

**Comment by jcsteh** [comment]
NVDA now requires (and enforces) Visual Studio 2015.
